vignettes/Cookbook.Rmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "Cookbook"
+author: "Jeremy Wildfire"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Cookbook}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Cookbook Vignette
+
+This vignette contains a series of examples showing how to initialize the safetyGraphics Shiny app in different scenarios. For a general overview of the app see [this vignette](). For more details about adding custom charts, see [this vignette]().
```

Comment:
It doesn't look like the 'this vignette' are linking to anything

Reply:
```suggestion
This vignette contains a series of examples showing how to initialize the safetyGraphics Shiny app in different scenarios. For a general overview of the app see [this vignette](Intro). For more details about adding custom charts, see [this vignette](chartConfiguration).
```